pipeline_steps/common.groovy

```diff
@@ -433,4 +433,48 @@ def is_doc_update_pr(git_dir) {
   return is_doc_update_pr
 }
 
+/* Look for JIRA issue key in commit messages for commits in the source branch
+ * that aren't in the target branch.
+ * This function uses environment variables injected by github pull request
+ * builder and so can only be used for PR triggered jobs
+ */
+def get_jira_issue_key(repo_path="rpc-openstack"){
```

Comment:
Commits can still reference multiple issues, but the primary issue should be the first part of the commit title, that will be the one that gets the build result comments. I think the github-jira integration will create a link from other jira issues that are mentioned in a commit message to the relevant PR, but I haven't tested that.